### ISSUE TITLE
Update to Roslyn RC1 and VS 2015 CTP 6

### ISSUE THREAD

**Issue by giggio** [code-cracker/code-cracker#288]

**Comment by jwooley** [comment]
It appears that this is required to run in VS 2015 CTP6. Should this be done in a separate branch or just push to master and force us to update. We're living on the bleeding edge anyway, so forcing upgrade may not be an issue. I'm not sure if you can version the nuget package for specific versions of VS to support multiple versions of the ide going forward (until RTM at least).

**Comment by giggio** [comment]
@jwooley Yes, this is bleeding edge. Next update, alpha4, will be out at the end of the month, and it will already be for CTP 6. I am assuming that anyone using Code Cracker at this moment is working on the bledding edge, or maybe only trying things out, and then they can use the current version (alpha3).

**Comment by jwooley** [comment]
I saw the list of API changes at https://github.com/dotnet/roslyn/wiki/VS-2015-CTP-6-API-Changes and suspect that this update is going to be non-trivial. If you would like help with the changes let me know. It might make sense to branch for this work if you need any help.

**Comment by carloscds** [comment]
@jwooley @giggio I already start to update project in my current PR.

**Comment by giggio** [comment]
I am about to finish removing errors. 16 to go on C#. Haven't look at VB yet.

**Comment by giggio** [comment]
Closed on 0c4a5a5.

**Comment by giggio** [comment]
There is work to do still. The solution is migrated, but now `CodeAction.Create` is not taking doc, project or solution anymore. I created a few `Task.FromResult` as a workaround, but there is need for a more detailed work, aligned to what was done on #273.
Build is disabled until AppVeyor updates their infrastructure to CTP6. I already contacted them yesterday. Build https://ci.appveyor.com/project/code-cracker/code-cracker/build/1.0.0.441 should not execute.
